.azure/templates/run-remote-performance-int.yml

```diff
@@ -0,0 +1,67 @@
+# This template contains steps to run performance tests for a single configuration.
+
+parameters:
+  config: 'Release'
+  arch: 'x64'
+  localTls: ''
+  remoteTls: ''
+  extraName: ''
+  extraArgs: '-Publish'
+
+jobs:
+- job: remote_performance_windows_${{ parameters.arch }}_${{ parameters.localTls }}_${{ parameters.remoteTls }}_${{ parameters.extraName }}
+  displayName: windows ${{ parameters.arch }} ${{ parameters.localTls }} ${{ parameters.remoteTls }} ${{ parameters.extraName }}
+  pool: MsQuic-Win-Perf
+  variables:
+  - name: runCodesignValidationInjection
+    value: false
+  - name: skipComponentGovernanceDetection
+    value: true
+  steps:
+  - checkout: self
+
+  - task: DeleteFiles@1
+    displayName: Clear Old Artifacts
+    inputs:
+      contents: '**/artifacts/*'
+
+  - template: ./download-artifacts.yml
+    parameters:
+      platform: windows
+      arch: ${{ parameters.arch }}
+      tls: ${{ parameters.localTls }}
+
+  - template: ./download-artifacts.yml
+    parameters:
+      platform: windows
+      arch: ${{ parameters.arch }}
+      tls: ${{ parameters.remoteTls }}
```

Comment:
You should be able to wrap this with a conditional and only execute it when `parameters.localTls` is `ne` to `parameters.remoteTls`.